packages/graphing/controller/src/index.js

```diff
@@ -0,0 +1,338 @@
+import debug from 'debug';
+
+import cloneDeep from 'lodash/cloneDeep';
+import lodash from 'lodash';
+import isEqual from 'lodash/isEqual';
+import { tools } from '@pie-lib/graphing';
```

Comment:
graphing is a ui package - you're using it in a controller. this is going to cause problems

if you need access to the logic in `utils` it should be in its own package with *no* UI dependencies.